src/components/presentational/Chat/Chat.css

```diff
@@ -0,0 +1,100 @@
+.mdhui-chat {
+    background: var(--mdhui-background-color-0);
+    font-size: 0.8em;
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    overflow: hidden;
+    min-height: 150px;
+}
+
+.mdhui-chat .mdhui-chat-log {
+    overflow: hidden;
+    flex: 1 1 0%;
+}
+
+.mdhui-chat .mdhui-chat-messages {
+    overflow-y: auto;
+    height: 100%;
+    width: 100%;
+}
+
+.mdhui-chat p,
+.mdhui-chat ul {
+    margin: .25rem 0;
+    word-break: break-word;
+}
+
+.mdhui-chat input,
+.mdhui-chat button {
+    min-height: var(--mdhui-touch);
+    min-width: var(--mdhui-touch);
+    border: 0;
+}
+
+.mdhui-chat .mdhui-chat-send-button {
+    position: absolute;
+    right: 4px;
+    cursor: pointer;
+    text-align: center;
+}
+
+.mdhui-chat input {
+    flex: 1;
+    font-size: 1em;
+    color: var(--mdhui-text-color-1);
+    padding: var(--mdhui-padding-md) var(--mdhui-touch) var(--mdhui-padding-md) var(--mdhui-padding-md);
+    box-sizing: border-box;
+    outline-offset: -1px;
+    background: var(--mdhui-background-color-0);
+}
+
+.mdhui-chat .mdhui-chat-input-group {
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    background: #fff;
+    border-top: 1px solid var(--mdhui-border-color-1);
+}
+
+.mdhui-chat .mdhui-chat-message,
+.mdhui-chat .mdhui-chat-message-loading {
+    padding: var(--mdhui-padding-xxs);
+    position: relative;
+}
+
+.mdhui-chat .mdhui-chat-received-message-row {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+}
+
+.mdhui-chat .mdhui-chat-sent-message-row {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    padding-right: var(--mdhui-padding-xs);
+}
+
+.mdhui-chat .mdhui-chat-sent-message {
+    background: var(--mdhui-background-color-1);
+    max-width: 70%;
+    padding: var(--mdhui-padding-xs) var(--mdhui-padding-sm);
+    border-radius: var(--mdhui-card-border-radius);
+}
+
+.mdhui-chat .mdhui-chat-message .svg-inline--fa,
+.mdhui-chat .mdhui-chat-message-loading .svg-inline--fa {
+    color: var(--mdhui-text-color-3);
+    margin-top: .35rem;
```

Comment:
I would definitely not use rems for this.  You ultimately want the text to be 16, 24, or 32px from the left edge of the screen.

My 2c is to drop the icon at this point?  The little flask icon I don't think works particularly great and it's kinda just jammed in there really close to the text.  And there is decent differentiation between the user messages and the AI messages due to the new border/background of the user messages

Either that or I would give the icon a little more room to breathe if we think we need it

Good rule of thumb is that no icons/text should be in the left or right 16px of the screen though

Reply:
This only affects the position of the icon with respect to the text on the right.

I added more left and right padding to the assistant messages. Replaced the icon with lightbulb as discussed.